sdks/python/src/opik/integrations/langchain/opik_tracer.py

```diff
@@ -1,12 +1,11 @@
 import logging
-from typing import TYPE_CHECKING, Any, Dict, List, Optional, Literal, Set
+from typing import Any, Dict, List, Literal, Optional, Set, TYPE_CHECKING
 
+from langchain_core.language_models.llms import BaseLLM
```

Comment:
```python
from langchain_core import language_models  # no need for llms since they add BaseLLM to language_models.__all__

language_models.BaseLLM.dict
```